e2e/test/scenarios/embedding/embedding-questions.cy.spec.js

```diff
@@ -212,6 +212,26 @@ describe("scenarios > embedding > questions", () => {
     // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
     cy.contains("October 7, 2023, 1:34 AM");
   });
+
+  it("should not crash on ignored HTTP status codes (1xx & 3xx) (metabase#51386)", () => {
+    cy.intercept("GET", "embed/question/**.csv**").as("dl");
+    cy.createQuestion(regularQuestion).then(({ body: { id } }) => {
+      cy.request("PUT", `/api/card/${id}`, { enable_embedding: true });
```

Comment:
Is the `x-metabase-client` header getting set here automatically, or do you need to do something with cy.request to send the custom header?

The "x-metabase-client" header should be either `"embedding-sdk-react"` or `"embedding-iframe"` in order to hit the proper codepath on BE.

Reply:
Let me focus on this rn

Reply:
So yes, they are, but I've added the headers as a part of the intercept to ensure that we're always checking on the right request